_posts/2023-05-09-april-rubygems-updates.md

```diff
@@ -0,0 +1,88 @@
+---
+title: April 2023 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in April.
+
+## RubyGems News
+
+This month in RubyGems, we released RubyGems [3.4.11](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3411--2023-04-10), [3.4.12](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3412--2023-04-11) and Bundler [2.4.11](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2411-april-10-2023), [2.4.12](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2412-april-11-2023).
+
```

Comment:
Since the last April release was at April 11th, most changes below are not actually included in those releases. I'd just mention the following:

> The main improvements that landed in our repository are:

> * Make Bundler fallback less to old APIs:  https://github.com/rubygems/rubygems/pull/6578.
> * Make RubyGems less dependent on default gems by using a vendored pure-ruby YAML parser to load its own configuration: https://github.com/rubygems/rubygems/pull/6615.
> * Some development improvements like RuboCop enhancements ([#6599](https://github.com/rubygems/rubygems/pull/6599), [#6608](https://github.com/rubygems/rubygems/pull/6608), [#6586](https://github.com/rubygems/rubygems/pull/6586), [#6590](https://github.com/rubygems/rubygems/pull/6590), [#6582](https://github.com/rubygems/rubygems/pull/6582)), or unifying our tasks to vendor gems ([#6628](https://github.com/rubygems/rubygems/pull/6628)).

Reply:
Thanks! I will update.